app/controllers/api/v1x0/requests_controller.rb

```diff
@@ -4,8 +4,16 @@ class RequestsController < ApplicationController
       include Mixins::IndexMixin
 
       def create
-        req = RequestCreateService.new(params.require(:workflow_id)).create(request_params)
-        json_response(req, :created)
+        if params[:request_id]
+          req = Request.find(params.require(:request_id))
+          raise Exceptions::ApprovalError, "Request has been processed!." unless req.current_stage
```

Comment:
`Request has finished its lifecycle. No more action can be added to its current stage.`